src/Wellcome.Dds/Wellcome.Dds.Dashboard/Controllers/DashController.cs

```diff
@@ -270,13 +269,12 @@ public async Task<ActionResult> Manifestation(string id)
                 }
                 return View("Manifestation", model);
             }
-            if (dgResource is IDigitisedCollection)
+            if (dgResource is IDigitisedCollection dgCollection)
```

Comment:
Nice - a new C# feature?

Reply:
Yes - C# pattern matching https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/is#type-pattern